gateway/src/main/kotlin/com/gitlab/kordlib/gateway/Intent.kt

```diff
@@ -148,14 +149,38 @@ sealed class Intent(val code: DiscordBitSet) {
      * - [TypingStart]
      */
     object DirectMessageTyping : Intent(1 shl 14)
+    companion object {
+        @OptIn(PrivilegedIntent::class)
+        fun values() = listOf(
+                DirectMessageTyping,
+                GuildIntegrations,
+                GuildEmojis,
+                DirectMessageTyping,
+                DirectMessages,
+                DirectMessagesReactions,
+                GuildBans,
+                Guilds,
+                GuildVoiceStates,
+                GuildMessages,
+                GuildMessageReactions,
+                GuildWebhooks,
+                GuildInvites,
+                GuildPresences,
+                GuildMembers
+        )
+    }
 }
 
 /**
  * A set of [intents][Intent] to be used while [identifying][Identify] a [Gateway] connection to communicate the events the client wishes to receive.
  */
 @Serializable(with = IntentsSerializer::class)
 data class Intents internal constructor(val code: DiscordBitSet) {
-
+    /**
+     *  Returns this [Intents] as a [Set] of [Intent]
+     */
+    @OptIn(PrivilegedIntent::class)
+    val intents = Intent.values().filter { it.code in code }.toSet()
```

Comment:
Don't make this a backing field property. I expect people to not interact with this all too much.